pages/10.contacts/01.manage-contacts/02.contact-monitoring/docs.en.md

```diff
@@ -288,21 +289,29 @@ Tracking codes support also [Google Analytics USERID][Google Analytics USERID] a
 
 There is a campaign action which allows you to send a custom event to [Google Analytics][Google Analytics] or [Facebook Pixel][Facebook Pixel] - it depends on there being a 'Visits a page' decision immediately before it in the campaign workflow.
 
-##### How to test Google Analytics tracking code and campaign action
+##### How to test Google Analytics tracking code and Campaign action
 
-- Install [Tag Assistant][Tag assistant] and enable recording on your website 
+- Install [Tag Assistant][Tag assistant] and enable recording on your website
 - Create campaign with the 'Visits a page' decision and 'Send tracking event' action 
 - Test it and check in the Tag Assistant debug window that you see one Pageview request and one event
 
-![Google Tag Assistant](google-analytics-tag-assistent.png)
+![Google Tag Assistant][google-analytics-tag-assistent.png]
+
+##### How to test Google Tag Manager tracking code and Campaign action
+
+- Install [Tag assistant][Tag assistant] and enable recording on your website
+- Create a Campaign with `Visits a Page` decision and `Send tracking event` action
+- Test it and see Tag assistant debug window with one Pageview request and 1 event.
+
+![Google Tag Assistant][google-analytics-tag-assistent.png]
```

Comment:
I think that we should add an explanation of what these options actually do:
![Configuration _ Mautic](https://user-images.githubusercontent.com/2930593/159236338-6ae1bcbd-ca6f-4567-a6e7-1520ae9e04bc.png)

Also I think it should say 'externally tracked pages' and 'mautic landing pages' which I will request in the PR.

Reply:
Done